### ISSUE TITLE
My game crashes at 22:00 6 jan 1964 while playing iberia... it happens always at that specifc time no matter what i do, it never gets past that. Eve when i reload, or start a new game altogether. My other game as russia, uses the exact same mods with no problems. I have no idea what could be wrong.

### ISSUE THREAD

**Issue by penedo0s** [TNODevs/TNO-Bug-Tracker#3168]
### Operating System

windows 11

### HOI4 version

1.14.4

### TNO version

v1.5.0

### Do you have other mods in use?

{"game":"hoi4","name":"TNO","mods":[{"displayName":"Cold War: The Iron Curtain - Music","enabled":true,"position":0,"steamId":"1364520259"},{"displayName":"Demand it! (Unlimited Ideologies)","enabled":true,"position":1,"steamId":"1498191141"},{"displayName":"Full Country Name Viewer","enabled":true,"position":2,"steamId":"2830876923"},{"displayName":"Mercenaries","enabled":true,"position":3,"steamId":"2279407933"},{"displayName":"No Division Limit","enabled":true,"position":4,"steamId":"1324280145"},{"displayName":"Rename States (with keyboard!)","enabled":true,"position":5,"steamId":"1689360351"},{"displayName":"Unique War Names","enabled":true,"position":6,"steamId":"2204846772"},{"displayName":"War Chess","enabled":true,"position":7,"steamId":"2936449085"},{"displayName":"WW3 and Cold War Music","enabled":true,"position":8,"steamId":"973566315"},{"displayName":"radio_tno","enabled":true,"position":9,"steamId":"2213783261"},{"displayName":"Historical Army Insignia","enabled":true,"position":10,"steamId":"2824225866"},{"displayName":"Cores & Claims GUI","enabled":true,"position":11,"steamId":"1692402147"},{"displayName":"Win for Warships - Updated Version","enabled":true,"position":12,"steamId":"2288508094"},{"displayName":"Precise Buffs - Cheat Ideas","enabled":true,"position":13,"steamId":"2600237628"},{"displayName":"Better Mechanics : Camera","enabled":true,"position":14,"steamId":"3009241301"},{"displayName":"Unit Sounds for Portugal","enabled":true,"position":15,"steamId":"2169302141"},{"displayName":"Better Mechanics  : Optimization","enabled":true,"position":16,"steamId":"3009437701"},{"displayName":"Better Mechanics : Combat","enabled":true,"position":17,"steamId":"2994638916"},{"displayName":"Better Mechanics : Frontline AI | Updated","enabled":true,"position":18,"steamId":"2984937408"},{"displayName":"Better Mechanics : Division Designer","enabled":true,"position":19,"steamId":"3010037573"},{"displayName":"Better Mechanics : Training and Experience","enabled":true,"position":20,"steamId":"3034632572"},{"displayName":"Better Mechanics : GFX Rework","enabled":true,"position":21,"steamId":"2938290655"},{"displayName":"Major Marches","enabled":true,"position":22,"steamId":"2425721100"},{"displayName":"Hearts of Iron: 1984 Music","enabled":true,"position":23,"steamId":"854408340"},{"displayName":"HOI4 - Texture Overhaul - Terrain","enabled":true,"position":24,"steamId":"1629702438"},{"displayName":"Minor Marches","enabled":true,"position":25,"steamId":"2476822613"},{"displayName":"Strategic View Adjustments","enabled":true,"position":26,"steamId":"704963980"},{"displayName":"The New Order: Cheat Mod++","enabled":true,"position":27,"steamId":"2606510574"},{"displayName":"The New Order: VT323 Map and Cities Font","enabled":true,"position":28,"steamId":"2665192983"},{"displayName":"TNO Achievements","enabled":true,"position":29,"steamId":"2895075046"},{"displayName":"TNO: ECONOMIC CHEAT","enabled":true,"position":30,"steamId":"2687000267"},{"displayName":"The New Order: Last Days of Europe","enabled":true,"position":31,"steamId":"2438003901"},{"displayName":"TNO: Loading Screens Redone","enabled":true,"position":32,"steamId":"2316340676"},{"displayName":"Toolpakt: Toolpack for TNO","enabled":true,"position":33,"steamId":"2700327893"},{"displayName":"Peace Deals","enabled":true,"position":34,"steamId":"2412445347"},{"displayName":"POW Mod","enabled":true,"position":35,"steamId":"3123481919"},{"displayName":"The New Order: Operation Deep Freeze","enabled":true,"position":36,"steamId":"2719053325"},{"displayName":"Based Radio","enabled":true,"position":37,"steamId":"2918309229"},{"displayName":"Cyberpunk 2077 Radio Stations","enabled":true,"position":38,"steamId":"3046801978"},{"displayName":"Disaster Radio Mod","enabled":true,"position":39,"steamId":"3035636462"},{"displayName":"Hotline Miami Radio","enabled":true,"position":40,"steamId":"2727263653"},{"displayName":"Phonk Radio","enabled":true,"position":41,"steamId":"3004573319"},{"displayName":"Total War Radio","enabled":true,"position":42,"steamId":"2281350437"},{"displayName":"Radio Through The Ages","enabled":true,"position":43,"steamId":"2103136236"},{"displayName":"Breaking Saul Music","enabled":true,"position":44,"steamId":"2806718893"},{"displayName":"Extremis Ultimis: Music Mod","enabled":true,"position":45,"steamId":"2981100489"},{"displayName":"Twilight of the Anthropocene Music Mod","enabled":true,"position":46,"steamId":"2996418512"},{"displayName":"Battleplan Execution Adjustments","enabled":true,"position":47,"steamId":"1905943555"},{"displayName":"Better Mechanics : Divisional AI","enabled":true,"position":48,"steamId":"2996504479"},{"displayName":"Better Mechanics : Urban Warfare","enabled":true,"position":49,"steamId":"3071265258"},{"displayName":"East vs West Station","enabled":true,"position":50,"steamId":"1842813819"},{"displayName":"HOI4 In Real Time","enabled":true,"position":51,"steamId":"2007026732"},{"displayName":"Vietnam Radio","enabled":true,"position":52,"steamId":"2557351552"},{"displayName":"Cheat Mods Combined","enabled":true,"position":53,"steamId":"843933244"},{"displayName":"Recall Volunteers","enabled":true,"position":54,"steamId":"2048998725"},{"displayName":"Rename Cities","enabled":true,"position":55,"steamId":"2847203943"},{"displayName":"Map Modes+","enabled":true,"position":56,"steamId":"2665267027"},{"displayName":"TNO Cheat Decisions","enabled":true,"position":57,"steamId":"2787480450"},{"displayName":"TNO修改器（Crys7aL_L1nE）","enabled":true,"position":58,"steamId":"2847981333"},{"displayName":"The New Order: Last Waltz Expanded","enabled":true,"position":59,"steamId":"2629037662"},{"displayName":"Better Mechanics : Elite Forces","enabled":true,"position":60,"steamId":"3152904675"},{"displayName":"Better Mechanics : War Exhaustion","enabled":true,"position":61,"steamId":"3132697120"},{"displayName":"+++Easybuff - The New Order","enabled":true,"position":62,"steamId":"3015271507"},{"displayName":"The New Order: Alt Flag Mod","enabled":true,"position":63,"steamId":"2875155897"},{"displayName":"TNO - Looong Official Names for Countries","enabled":true,"position":64,"steamId":"3128795384"},{"displayName":"TNO: Allegiances Submod","enabled":true,"position":65,"steamId":"2897376278"},{"displayName":"Interactive Piano For HOI4","enabled":true,"position":66,"steamId":"2217515723"},{"displayName":"Interactive Game Boy for TNO","enabled":true,"position":67,"steamId":"2209436216"},{"displayName":"Diplomacy Alignment Triangle for The New Order","enabled":true,"position":68,"steamId":"2183573918"}]}

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Files

[error.log](https://github.com/TNODevs/TNO-Bug-Tracker/files/14997216/error.log)

### Describe the crash

Basically, the game crashes altogether in 22:00, 6 jan 1964. I've tried redoing saves, starting a new game, none of it works, when i reach this date, it dies, takes the game out, and gives the error message. What is peculiar, is that i have another game, wirh the exact same mods, and it works fine. I have no idea why it woudnt here.

### Steps to reproduce / Cause of the crash

1. well, put all the mods i have until this date, it should crash then

**Comment by penedo0s** [comment]
i look forward to your response

**Comment by Jack121Q** [comment]
Honestly, if it works fine in other games, chances are your save-file got corrupted. Nothing we can do about that sadly.
